### DISCUSSION TITLE
Foster Equation

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8440] [Ideas]
The Foster Equation is a mission chain set to happen after you get the mission that requires you to take a wood chip from The Deep (The mission just needs to be offered, and if the player declines then the mission chain will have less options, though nothing important changes). 

Robert Foster was a very brilliant scientist, well loved albeit a bit eccentric. Robert was one of the minds who fully understood the Hyperdrive, not an easy task. His inventions helped the entire Republic and Syndicate, and he was soon going to be awarded a monetary prize for his services to humankind. It only added to the shock when Robert disappeared. Before he disappeared, he gave a hint that he had an equation that could change humankind again. A day later, his wife reported his absence. The Republic mobilized a hunt for the "Foster Equation", but had little success. The Foster Equation was forgotten, and the Republic soon discontinued the mission.

Until Albert Foster moved out of one of the Foster manors (a generous claim, as they were pretty much above average 3 story houses), there was almost no hope for finding the Foster Equation. But as he was packing, a note slipped out. "R.F-Equation. If you have this note and are not an Equation Hunter, then leave this note here. If you search for the now called 'Foster Equation', then pay close heed to my words." The following words are undecipherable. Maybe it is a possible dialect of an alien race, like the Quarg?

This is just a rough idea that could introduce advances to the human race, or possibly a way to communicate to other species effectively.

**Comment by Zitchas** [comment]
Interesting idea.

**Comment by Luuiscool4567** [comment]
I agree with @Zitchas. This has potential to be a great side plot after Hai Reveal or something like that. It would be nice to see a Core Archaeology or something like that.
